### PR TITLE
Upgrade Flutter to v3.10.5

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

**Comment by Arsnael** [comment]
So:
- need https://github.com/dab246/super_tag_editor/pull/21 to be merged and new version 0.1.4 being published
- need https://github.com/linagora/flutter-date-range-picker/pull/5 to be merged on master

As both those libs are Linagora (well one is on @dab246 personal github actually)

By pointing to those libs modified locally as their respective PR, the ./prebuild.sh compile with Flutter 3.10.5.

I still have work to do as when I try to run it for Android emulator there is errors with code to fix. So it's still a WIP

File: pubspec.yaml [hunk 1]
```diff
@@ -189,7 +189,7 @@ dependencies:
   # flutter_localizations depends on intl 0.18.0
   intl: 0.18.0
 
-  super_tag_editor: 0.1.3
+  super_tag_editor: 0.1.4
```
Comment (Arsnael) [inline pubspec.yaml:192]: Needs this https://github.com/dab246/super_tag_editor/pull/21 to be merged and published first

**Comment by hoangdat** [comment]
I propose to merge to branch `upgrade_3_10_x`, and test all the thing on it before merge to `master` branch. Thanks

**Comment by Arsnael** [comment]
cc @dab246 @hoangdat 

I can:
- compile with prebuild.sh
- run the app with an android emulator with `flutter run`
- build and run tmail-web

I do not have a macbook so I can't check for iOS app though. I didn't test too much though.

Of course the last commit should be removed when the 2 other projects have been merged/released with necessary changes for upgrading to flutter 3.10.5.

If any remarks or feedback, don't hesitate :)

**Comment by tk-nguyen** [comment]
FYI, just ignore the failing `Deploy PR on Pages` check. Because this is running from a fork, so this workflow cannot access secrets.

**Comment by dab246** [comment]
I will close this PR. Please follow the full PR upgrade at #2036

**Pull request closed without merging**